go.mod

```diff
@@ -18,7 +18,7 @@ require (
 	github.com/hashicorp/terraform-exec v0.17.3
 	github.com/hashicorp/terraform-json v0.14.0
 	github.com/hashicorp/terraform-registry-address v0.0.0-20220623143253-7d51757b572c
-	github.com/hashicorp/terraform-schema v0.0.0-20221018140729-4380e150ec9f
+	github.com/hashicorp/terraform-schema v0.0.0-20221028152941-ef7f11b3758a
```

Comment:
This hash corresponds to a different commit than you probably intended?
https://github.com/hashicorp/terraform-schema/commit/ef7f11b3758a

FWIW I usually use `go get github.com/hashicorp/terraform-schema@latest` - the `latest` keyword _should_ pull down the latest version or revision in most cases.